content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -207,13 +207,18 @@ starts an injected pod with the following steps:
 Init containers execute before the sidecar proxy starts, which can result in traffic loss during their execution.
 Avoid this traffic loss with one or both of the following settings:
 
+* Set the user of init container to `1337` using `runAsUser`. This will make traffic from init container bypass Istio iptables capture.
+  Application container traffic will still be captured as usual.
 * Set the `traffic.sidecar.istio.io/excludeOutboundIPRanges` annotation to disable redirecting traffic to any
   CIDRs the init containers communicate with.
 * Set the `traffic.sidecar.istio.io/excludeOutboundPorts` annotation to disable redirecting traffic to the
```

Comment:
```suggestion
# Set the uid of the init container to `1337` using `runAsUser`. This will make traffic from the init container bypass Istio's `iptables` capture.
  Application container traffic will still be captured as usual.
# Set the `traffic.sidecar.istio.io/excludeOutboundIPRanges` annotation to disable redirecting traffic to any
  CIDRs the init containers communicate with.
# Set the `traffic.sidecar.istio.io/excludeOutboundPorts` annotation to disable redirecting traffic to the
```

changing bullets to numbers as you're saying "only #1 will work" below